index.d.ts

```diff
@@ -15,6 +15,26 @@ declare namespace cpFile {
 		@default 0o777
 		*/
 		readonly directoryMode?: number;
+
+		/**
+		Callback function for progress reporting.
```

Comment:
I wrote the same sentence, and got the below feedback.

Probably it applies to this too.

https://github.com/sindresorhus/del/pull/141#discussion_r852223113

Reply:
I've tried to reformulate.